### PR TITLE
Added Ocean's 8

### PR DESCRIPTION
**Pull request by igor-nm**
Ocean's 8 movie in json file

### CODE REVIEW

**Comment by igor-nm** [comment]
From these updates #78

File: movies/2018/ocean-8.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        "Rihanna",
+        "Helena Bonham Carter"
+    ],
+    "plot": "Debbie Ocean gathers an all-female crew to attempt an impossible heist at New York City's yearly Met Gala.",
+    "future": true,
+    "release-date": "2018-06-08",
+    "categories": [
+        "Crime",
+        "Comedy",
+        "Thriller"
```
Comment (leonlafa) [inline movies/2018/ocean-8.json]: need to be lowercase && remove Crime

File: movies/2018/ocean-8.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Ocean's 8",
```
Comment (leonlafa) [inline movies/2018/ocean-8.json]: title of file should be oceans-8

File: movies/2018/ocean-8.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        "Sandra Bullock",
+        "Cate Blanchett",
+        "Anne Hathaway",
+        "Mindy Kaling",
+        "Sarah Paulson",
+        "Awkwafina",
+        "Rihanna",
+        "Helena Bonham Carter"
+    ],
+    "plot": "Debbie Ocean gathers an all-female crew to attempt an impossible heist at New York City's yearly Met Gala.",
```
Comment (leonlafa) [inline movies/2018/ocean-8.json]: should be storyline.

Comment (leonlafa) [reply movies/2018/ocean-8.json]: Screenshot 2019-10-10 at 15 27 14
 Please can you make your json follow the same structure as ^

File: movies/2018/ocean-8.json (hunk 1 above)
Comment (igor-nm) [reply movies/2018/ocean-8.json]: As request, it was solved.

File: movies/2018/ocean-8.json (hunk 2 above)
Comment (igor-nm) [reply movies/2018/ocean-8.json]: As request, it was solved.

File: movies/2018/ocean-8.json (hunk 3 above)
Comment (igor-nm) [reply movies/2018/ocean-8.json]: As request, it was solved.

**Comment by igor-nm** [comment]
Issue reference #78

**Pull request merged**